media-tv/mythtv/Manifest

```diff
@@ -1,2 +1,2 @@
-DIST mythtv-29.1-r1.tar.gz 105657217 BLAKE2B 9d42464b0c75c2d3a359b528e3f7917cfb281f48451167afc6517caa9702da44fce2d48566fd902ec08af35117232113b4d499e2c9a3cf2ad2634d91cf532ca7 SHA512 0eadc37661d9fa93a63c0f425cd43d43533889b0afcea1478b2645e083ba6c96eebf2eec088db1ed6f0a0727c03b7c90c0c18b83c194d630dda3bac45bca5d87
+DIST mythtv-29.1-r2.tar.gz 105657217 BLAKE2B 9d42464b0c75c2d3a359b528e3f7917cfb281f48451167afc6517caa9702da44fce2d48566fd902ec08af35117232113b4d499e2c9a3cf2ad2634d91cf532ca7 SHA512 0eadc37661d9fa93a63c0f425cd43d43533889b0afcea1478b2645e083ba6c96eebf2eec088db1ed6f0a0727c03b7c90c0c18b83c194d630dda3bac45bca5d87
```

Comment:
This should be `${P}`, as you can see, the manifest hash is the same, so users are downloading the same massive file again. Also it stresses the mirrors. ALSO dont use `git mv` but you need to separate commits for addition and removal. But please, for now, make all 29.1 ebuilds use the -r1.tar.gz so it doesnt need to be downloaded and re-hosted again.

Reply:
make all 29.1 ebuilds use the -r1.tar.gz

Reply:
Restored mythtv-29.1-r1.ebuild in case someone needs it.